### PR TITLE
Ember CLI Upgrade/Improvements

### PR DESCRIPTION
**Pull request by blimmer**

### CODE REVIEW

**Comment by thoov** [comment]
Thanks!

**Comment by blimmer** [comment]
No problem! Might want a version bump too, which I'm happy to do on this branch. `1.0.1` seems right.

**Comment by thoov** [comment]
Normally you don't want to do version bumps on the same branch as the changes.

**Comment by thoov** [comment]
So we can do that after this gets merged

File: package.json [hunk 1]
```diff
@@ -7,41 +7,42 @@
...
     "build": "ember build",
-    "test": "ember test"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/stefanpenner/ember-state-services.git"
+    "start": "ember server",
+    "test": "ember try:testall"
   },
+  "repository": "",
```
Comment (thoov) [inline package.json]: Can you add back the repo

Comment (blimmer) [inline package.json]: Whoops, added back.

On Fri, Aug 14, 2015 at 10:41 AM Travis Hoover <notifications@github.com>
wrote:

> In package.json
> <https://github.com/stefanpenner/ember-state-services/pull/26#discussion_r37094281>
> :
>
> >    },
> > +  "repository": "",
>
> Can you add back the repo
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stefanpenner/ember-state-services/pull/26/files#r37094281>
> .
>
-- 
[sent from my mobile]

**Comment by stefanpenner** [comment]
:+1:

**Pull request merged**